pkg/epforwarder/epforwarder.go

```diff
@@ -69,6 +70,18 @@ var passthroughPipelineDescs = []passthroughPipelineDesc{
 		defaultBatchMaxSize:           pkgconfig.DefaultBatchMaxSize,
 		defaultInputChanSize:          pkgconfig.DefaultInputChanSize,
 	},
+	{
+		eventType:              eventTypeDBMMetadata,
+		contentType:            http.JSONContentType,
+		endpointsConfigPrefix:  "database_monitoring.metadata.",
```

Comment:
This works fine for me either way but I wonder if it would be time to just converge on the dbm-metrics-intake and have this be `database_monitoring.metrics.` instead.

Reply:
yeah we can do that too. It would simplify the PR. I'll do that since it looks like i have some failing tests & we really do not need to create an extra config for this data

Reply:
OK, done 👍